grafast/grafserv/package.json

```diff
@@ -96,6 +98,7 @@
     }
   },
   "devDependencies": {
+    "@types/aws-lambda": "^8.10.122",
```

Comment:
Is it right that the types are v8 but the library is v1?

Reply:
`@types/aws-lambda` is right and `aws-lambda` is a totally unrelated 3rd party package smh

https://docs.aws.amazon.com/lambda/latest/dg/typescript-handler.html